app/Http/Controllers/Backend/User/TokenController.php

```diff
@@ -29,6 +31,22 @@ public static function revokeToken(string $tokenId, User $user): void {
         }
 
         $token->revoke();
+        $client = $token->client()->first();
+        if ($client && $client->webhooks_enabled) {
```

Comment:
Was passiert, wenn der Client das `webhooks_enabled` flag an hatte, damit Webhooks registriert hatte und dann den Flag entfernt hat? Werden die bestehenden Webhooks dann entfernt? (Wahrscheinlich Frage für @NyCodeGHG?)

Was ich sagen will, ist: 
```suggestion
        if ($client) {
```

Reply:
Die werden bisher nicht entfernt, wäre vermutlich die bessere Lösung diesen Flag nicht deaktivierbar zu machen wenn Webhooks existieren

Reply:
Habe ich in f1637a5c90ba2d420b633b62d3d34e37772b2e73 umgesetzt